src/components/styled-components/StyledTodoList.tsx

```diff
@@ -0,0 +1,7 @@
+import { styled } from 'styled-components';
```

Comment:
unify import like you did in other places:

`import styled from 'styled-components';`